libm/fplib/Files.am

```diff
@@ -106,7 +106,6 @@ libm_a_asm_sources = \
 	unordsf2.S
 
 libm_a_extra_dist = \
-	asmdef.h \
-	fp32def.h 
+	asmdef.h
```

Comment:
Why is `fp32def.h` being removed from the distro? Typo?

Reply:
Yeah, that's a typo. I only remember removing `ntz.h` and `asmdef.h` from `Files.am`.

Reply:
Changed that to remove `asmdef.h` instead.